borg/helpers.py

```diff
@@ -879,7 +929,13 @@ def remove_surrogates(s, errors='replace'):
 def make_path_safe(path):
     """Make path safe by making it relative and local
     """
-    return _safe_re.sub('', path) or '.'
+    if sys.platform != 'win32':
+        return _safe_re.sub('', path) or '.'
+    else:
+        tail = path
+        if path[0:2] == '//' or path[0:2] == '\\\\' or path[1] == ':':
+            drive, tail = os.path.splitdrive(path)
```

Comment:
well, they are relative as the leading slash is missing, but they are still full pathes.

if we omit the drive letter, information about where it came from is lost.

an idea i had an hour ago was "maybe we should not initially support drive letters or backslashes", so all the duplicated code isn't necessary. you can still change to d: before doing a backup.